umm-lib/src/cmr/umm/umm_core.clj

```diff
@@ -1,24 +1,28 @@
 (ns cmr.umm.umm-core
   "Functions to transform concepts between formats."
-  (:require [cmr.common.mime-types :as mt]
-            [cmr.common.xml :as cx]
-            [cmr.umm.echo10.echo10-core :as echo10]
-            [cmr.umm.echo10.echo10-collection :as echo10-c]
-            [cmr.umm.echo10.granule :as echo10-g]
-            [cmr.umm.dif.dif-core :as dif]
-            [cmr.umm.dif10.dif10-core :as dif10]
-            [cmr.umm.dif.dif-collection :as dif-c]
-            [cmr.umm.dif10.dif10-collection :as dif10-c]
-            [cmr.umm.iso-mends.iso-mends-core :as iso-mends]
-            [cmr.umm.iso-mends.iso-mends-collection :as iso-mends-c]
-            [cmr.umm.iso-mends.granule :as iso-mends-g]
-            [cmr.umm.iso-smap.iso-smap-core :as iso-smap]
-            [cmr.umm.iso-smap.iso-smap-collection :as iso-smap-c]
-            [cmr.umm.iso-smap.granule :as iso-smap-g]
-            [cmr.umm.umm-collection]
-            [cmr.umm.umm-granule])
-  (:import cmr.umm.umm_collection.UmmCollection
-           cmr.umm.umm_granule.UmmGranule))
+  (:require
```

Comment:
We don't need to change `umm-lib` for new concept types. New concept types should only be added in `umm-spec-lib`.